stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/EventHandlerImpl.kt

```diff
@@ -431,15 +431,12 @@ internal class EventHandlerImpl(
                     domainImpl.setOffline()
                 }
                 is ConnectedEvent -> {
-                    val recovered = domainImpl.isInitialized()
-
                     domainImpl.setOnline()
                     domainImpl.setInitialized()
-                    if (recovered && domainImpl.recoveryEnabled) {
-                        domainImpl.connectionRecovered(true)
-                    } else {
-                        domainImpl.connectionRecovered(false)
-                    }
+                    domainImpl.connectionRecovered(domainImpl.recoveryEnabled)
```

Comment:
the new code here doesn't differentiate between first connect and recovery flow

Reply:
If it is first connect, there wasn't any pending channel, neither anyy calls to be done on the server